_rules/button-non-empty-accessible-name-97a4e1.md

```diff
@@ -39,12 +39,12 @@ This rule considers an exception for "image buttons" (i.e., `input` elements wit
 
 - The rule assumes that all buttons are [user interface components as defined by WCAG 2](https://www.w3.org/TR/WCAG22/#dfn-user-interface-components).
 
+- Some elements have a role of `button` and a default accessible name defined by the [HTML Accessibility API Mapping][html aam input button], for example `input` elements whose `type` [attribute value][] is either `submit` or `reset`. This rule considers that these default names can be descriptive and therefore does not fail them.
```

Comment:
should the HTML Accessibility API Mapping link be updated? It seems it's currently not referencing anything and goes to the whole file instead, new link: https://www.w3.org/TR/html-aam-1.0/#input-type-button-input-type-submit-and-input-type-reset-accessible-name-computation

The same issue is in the "Bibliography" section of the rule